src/lib/index.ts

```diff
@@ -65,6 +118,26 @@ export class MPlayerMediaItem {
     }, DEFAULT_OP_TIMEOUT);
   }
 
+  /**
+   * getMeta
+   * 
+   * @returns a promise that resolves with the metadata
+   */
+  public getMeta(): Promise<Metadata> {
```

Comment:
This is on the public interface - I would prefer the name to be `getMetadata()`

Reply:
:+1: